lxcfs.c

```diff
@@ -489,6 +497,10 @@ static int lxcfs_readdir(const char *path, void *buf, fuse_fill_dir_t filler, of
 static int lxcfs_access(const char *path, int mode)
 {
 	int ret;
+
+	if (strcmp(path, "/") == 0 && access(path, R_OK) == 0)
```

Comment:
I don't think we need this - the admin is not meant to change perms on /var/lib/lxcfs itself to deny read access.  what I meant was checking that the mode that is passed in does not contain write.